pkg/controller/volume/persistentvolume/pv_controller_base.go

```diff
@@ -245,20 +250,29 @@ func (ctrl *PersistentVolumeController) updateClaim(claim *v1.PersistentVolumeCl
 	}
 }
 
+// Unit test [5-5] [5-6] [5-7]
 // deleteClaim runs in worker thread and handles "claim deleted" event.
 func (ctrl *PersistentVolumeController) deleteClaim(claim *v1.PersistentVolumeClaim) {
 	_ = ctrl.claims.Delete(claim)
-	klog.V(4).Infof("claim %q deleted", claimToClaimKey(claim))
+	claimKey := claimToClaimKey(claim)
+	klog.V(4).Infof("claim %q deleted", claimKey)
+	// clean any possible unfinished provision/prebinding start timestamp from cache
+	// Unit test [5-8] [5-9]
+	provisionTsKey := ctrl.createOperationTimestampCacheKey("provision", claimKey)
```

Comment:
Can we also have a delete/abort wrapper?

Reply:
done